po/fvwmpo.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
+FVWMPOT=fvwm3.pot
+MSGINIT=/usr/bin/msginit
+MSGFMT=/usr/bin/msgfmt
+MSGUNIQ=/usr/bin/msguniq
+MSGMERGE=/usr/bin/msgmerge
+XGETTEXT=/usr/bin/xgettext
+
```

Comment:
I know this is lifted from the Makefile, but I think we should drop the path-prefixes to the binaries, and let $PATH handle these.

Especially when you consider gettext on *BSD is likely to use /usr/local/bin as the location.

Reply:
Full paths have been removed.